azure-pipelines.yml

```diff
@@ -6,6 +6,7 @@ pr:
 - main
 - feature/*
 - release/*
+- imiles/*
```

Comment:
This change is just for testing the e2e builds, not for the final update that will go to the main branch, right?

Reply:
That's right, it's removed now. 🙂